Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -1815,9 +1816,7 @@ private void SetMaterials()
             {
                 float[] color = { 1.0f, 1.0f, 1.0f, 0.75f };
 
-                Shader shader = Shader.Find("Mixed Reality Toolkit/Standard");
-
-                wireframeMaterial = new Material(shader);
+                wireframeMaterial = new Material(StandardShaderUtility.MrtkStandardShader);
```

Comment:
I was just thinking about this. So many inspector classes default to MRTK shader. Great to have it in one place!